doc/user/content/guides/reuse-topic-for-kafka-sink.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Topic reuse for Kafka sinks"
```

Comment:
I prefer "Kafka sink topic reuse", since the subject is only relevant to Kafka users, so we should lead with Kafka at the beginning

Reply:
What about using "Exactly-once Kafka sinks"? As raised in [this (internal) thread](https://materializeinc.slack.com/archives/CQM08AP9V/p1627892841002100), this might be more intuitive for users coming from other systems, and we don't mention it at all in the docs (@msudberg).

Reply:
I have no opinion on if the top line term should be topic reuse or exactly once - but we should say exactly once somewhere in the docs because it is a common term in the ecosystem.

Reply:
I've added a line about exactly-once sinks to both this guide and the CREATE sink page. I think the title should reflect how we currently talk about the topics in the docs, though, so I've just followed @awang's suggestion of "Kafka Sink Topic Reuse" for the title. I added "exactly-once sinks" to the topic description, which I believe will help with search results.